packages/sails-ng-common/src/validation/form.model.ts

```diff
@@ -1,12 +1,16 @@
+import { get as _get } from 'lodash';
+
 /**
  * The map of validation errors.
  *
  * Typically, has a property whose name is the validation key, e.g. 'min', and
  * value is a dictionary of message and params that are arbitrary values
  * that can be used to render an error message template.
+ *
+ * This is similar to the angular form ValidationErrors type.
  */
 export type FormValidatorErrors = {
-  [key: string]: { message: string; params: { [key: string]: unknown } };
+  [key: string]: any;
```

Comment:
This has probably been covered but I've forgotten. What's the reason we're switching this to any?

Reply:
I changed this to match the angular [ValidationErrors](https://angular.dev/api/forms/ValidationErrors), but we know more about what structure the errors should have, so I'll change this back to a specific type.

Reply:
I've made the types clearer and more strict in 4ea1e52